plugins/ccgx/fu-ccgx-dmc-common.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-ccgx-dmc-common.h"
+
+gchar *
+fu_ccgx_dmc_version_to_string (guint32 val)
```

Comment:
Isn't this just `fu_common_version_from_uint32(FWUPD_VERSION_FORMAT_QUAD)`?

Reply:
Replaced it with fu_common_version_from_uint32 on commit today.